rules/imports.js

```diff
@@ -0,0 +1,159 @@
+module.exports = {
+  plugins: ['import'],
```

Comment:
Nit, but in `index.js` we use the fully justified plugin names (`eslint-plugin-import`). No strong preference either way, but slight preference for consistency.